src/python/pants/backend/nfpm/native_libs/elfdeps/BUILD

```diff
@@ -0,0 +1,19 @@
+# Copyright 2025 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+resources(name="lockfile", sources=["elfdeps.lock*"])
+
+python_sources(
+    overrides={
+        "subsystem.py": dict(
+            dependencies=[
+                ":lockfile",
+                "./analyze.py",
```

Comment:
This pulls in `analyze.py` as a python dependency, which should also pull in a transitive elfdeps dep (added by dependency inference).

Instead, I could make this depend on resource target. Doing that will mean the file is owned by multiple targets. Should I something like this?

```python
resources(name="scripts", sources=["analyze.py"])
```

Reply:
That worked very nicely. I now have a `resources` target for `subsystem.py` to depend on. And I now have `__dependents_rules__` to prevent the pants wheel+pex from depending on this script as a `python_source`.